chef/cookbooks/ceilometer/recipes/server_ha.rb

```diff
@@ -60,9 +60,17 @@
 if node[:ceilometer][:use_mongodb]
   pacemaker_order "o-ceilometer-mongo" do
     score "Mandatory"
-    ordering "cl-mongodb cl-g-ceilometer-server"
+    ordering "cl-mongodb cl-#{group_name}"
+    action :create
     only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
   end
 end
 
+crowbar_pacemaker_order_only_existing "o-cl-#{group_name}" do
+  ordering [ "rabbitmq", "cl-keystone", "cl-#{group_name}" ]
```

Comment:
Haven't put postgresql as I'm unsure if it should be optional or mandatory (like for mongodb). Any opinion,